### ISSUE TITLE
Crossbow rendering in third person is weird

### ISSUE THREAD

**Issue by Zsashas** [SlimeKnights/TinkersConstruct#3265]
First image is with a bolt loaded, second is without.

Also, will we ever see a return of the fancy first-person crossbow model from 1.7.10?

![2017-12-08_01 48 49](https://user-images.githubusercontent.com/6917535/33760593-09afc5b0-dbbb-11e7-9326-562b69d9e94a.png)
![2017-12-08_01 49 08](https://user-images.githubusercontent.com/6917535/33760595-09f34cae-dbbb-11e7-9da4-414cddcb872d.png)

**Comment by KnightMiner** [comment]
Duplicate of #2641, we really need to get around to fixing that.

**Comment by bonii-xx** [comment]
Well, that's something everybody can fix easily since it's pure json-model-stuff! Maybe some volunteer wants to do it ;o

**Comment by spookyjuke** [comment]
Mind writing up some detailed instructions on how to do that? I would like to help fix it because it looks absolutely silly right now in third person but I'm assuming by everybody you mean everybody who is familiar with json model stuff or development. I would have no idea how to start it.

**Comment by KnightMiner** [comment]
[Check out the article on the Minecraf Wiki](http://minecraft.gamepedia.com/Model). It describes the basics of the model format and model display tags.

**Comment by spookyjuke** [comment]
OK I did some playing around and fixed up those attachment points for third person.
![crossbow third person](https://user-images.githubusercontent.com/23448033/39732517-e5c39ed6-523b-11e8-94c9-371a0891e54b.gif)

Then just for shits I checked on the shortbow/longbow and those were... not right either so I fixed those up after some trial and error and comparison with the vanilla bow's json.

![sblb](https://user-images.githubusercontent.com/23448033/39732580-40002630-523c-11e8-8b2d-6092dc1affb4.jpg)

Only thing I couldn't figure out was how to get the arrows to not be visible while simply holding a bow in your hand undrawn. Seems like a mistake to me. Besides that, I've now got these 3 fixed json files ready. I'm not exactly a github guy so where exactly can I post these so that the developer can put them in the releases?
